pkg/api-server/testdata/resources/inspect/dataplanes/_rules/meshhttproute.golden.json

```diff
@@ -40,7 +40,15 @@
          "backendRefs": [
           {
            "kind": "MeshServiceSubset",
-           "name": "backend_kuma-demo_svc_3001",
+           "name": "other-svc",
```

Comment:
why did the order change here?

Reply:
We were missing sorting by name, fixed

Reply:
Seems like it's still different order.It's because we were sorting by nodes ids and not string name. I think it's just best to keep order of names in tags